elliottlib/util.py

```diff
@@ -31,6 +31,9 @@ def green_print(msg):
     """Print out a message in green text"""
     click.secho(msg, nl=True, bold=False, fg='green')
 
+def yellow_print(msg):
+    """Print out a message in yellow text"""
+    click.secho(msg, nl=True, bold=False, fg='yellow')
```

Comment:
```suggestion
    click.echo(msg, nl=True, bold=False, fg='yellow')
```

Reply:
click.secho(x) is an alias to click.echo(click.style(x)).

http://click.palletsprojects.com/en/5.x/utils/

Reply:
shows what i know!